app/components/typography/DescriptionWithChildren.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+interface DescriptionWithChildrenProps {
+  text: string;
+  lineHeight: string;
+  letterSpacing?: string;
```

Comment:
Validate props like lineHeight and letterSpacing by restricting them to specific CSS classes to avoid runtime errors or inconsistencies.
```suggestion
  lineHeight: "leading-normal" | "leading-loose" | "leading-tight"; // Restrict values
  letterSpacing?: "tracking-normal" | "tracking-wide" | "tracking-tight"; // Restrict values
```

Reply:
done